### DISCUSSION TITLE
Full CPU load on Firefox with simple sample code

### DISCUSSION THREAD

**Discussion by skoe** [DioxusLabs/dioxus#1928] [Q&A] [answered]
I'm starting with dioxus-web and the sample code from https://dioxuslabs.com/learn/0.4/getting_started/wasm. Dioxus is 0.4.1.

I run it with dx serve and open the page with Firefox: "Hello, world!" appears and the CPU runs at 100 % (or 25 % on my quad core system :) ). The system monitor on my Ubuntu shows that "Isolated Web content" is the reason. The console of the browser says: "Firefox can’t establish a connection to the server at ws://localhost:8080/_dioxus/hot_reload." This may or may not be related to my issue.

With Chrome I don't get this high CPU load.

What could be the reason? As I'm quite new to all this Web stuff I don't know where to look next.

**Comment by ealmloff** [comment, marked as answer]
That sounds like the same issue as https://github.com/DioxusLabs/dioxus/pull/1925. You could try running with hot reloading enabled in the CLI (`dx serve --hot-reload`), disable the (default enabled) hot reload feature on the web renderer, or run in release mode

**Reply by skoe** [reply]
Thanks for your help, --hot-reload works. Still much magic under the hood that I don't understand, but I'm looking forward to gain deeper insight :) Thanks again!
